addons/content-index/src/main/java/org/commonjava/indy/content/index/IndexingContentManagerDecorator.java

```diff
@@ -257,6 +257,25 @@ private Transfer getIndexedTransfer( final StoreKey storeKey, final StoreKey top
         Logger logger = LoggerFactory.getLogger( getClass() );
         logger.trace( "Looking for indexed path: {} in: {} (entry point: {})", path, storeKey, topKey );
 
+        try
+        {
+            ArtifactStore store = storeDataManager.getArtifactStore( storeKey );
+            if ( store.isDisabled() )
+            {
+                logger.info( "Content not available in index caching layer due to store disabled for {} in group {}",
+                             storeKey, topKey );
+                return null;
+            }
+        }
+        catch ( IndyDataException e )
+        {
+            logger.error(
+                    String.format( "Failed to lookup store: %s (in membership of: %s). Reason: %s", storeKey, topKey,
+                                   e.getMessage() ), e );
+            //FIXME: Should this be null?
```

Comment:
I think we need to be throwing IndyWorkflowException here, to be consistent with:

https://github.com/commonjava/indy/blob/a1a9b633f8dfb3b423666f69e5e4d776e7dd18a4/core/src/main/java/org/commonjava/indy/core/content/DefaultContentManager.java#L394-L394